handler/oauth2/flow_resource_owner_test.go

```diff
@@ -111,6 +113,17 @@ func TestResourceOwnerFlow_PopulateTokenEndpointResponse(t *testing.T) {
 				store.EXPECT().CreateAccessTokenSession(nil, "bar", areq).Return(nil)
 			},
 		},
+		{
+			description: "should pass - offline scope",
+			setup: func() {
+				areq.GrantTypes = fosite.Arguments{"password"}
+				areq.GrantScope("offline")
+				rtstr.EXPECT().GenerateRefreshToken(nil, areq).Return("tokenfoo.bar", "bar", nil)
+				store.EXPECT().CreateRefreshTokenSession(nil, "bar", areq).Return(nil)
+				chgen.EXPECT().GenerateAccessToken(nil, areq).Return("tokenfoo.bar", "bar", nil)
+				store.EXPECT().CreateAccessTokenSession(nil, "bar", areq).Return(nil)
+			},
```

Comment:
Good idea; I'll add to the tests to check for `refresh_token` with and without the `offline` scope.